[2024-12-26 18:41:46,499] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_three 2024-12-26T18:40:17.282029+00:00 [None]>
[2024-12-26 18:41:46,514] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_three 2024-12-26T18:40:17.282029+00:00 [None]>
[2024-12-26 18:41:46,515] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-12-26 18:41:46,516] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-12-26 18:41:46,517] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-12-26 18:41:46,526] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): task_three> on 2024-12-26T18:40:17.282029+00:00
[2024-12-26 18:41:46,530] {standard_task_runner.py:52} INFO - Started process 1069 to run task
[2024-12-26 18:41:46,533] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'assignment', 'task_three', '2024-12-26T18:40:17.282029+00:00', '--job-id', '33', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/assignment.py', '--cfg-path', '/tmp/tmp5unggq18', '--error-file', '/tmp/tmprnk14mpn']
[2024-12-26 18:41:46,535] {standard_task_runner.py:77} INFO - Job 33: Subtask task_three
[2024-12-26 18:41:46,569] {logging_mixin.py:104} INFO - Running <TaskInstance: assignment.task_three 2024-12-26T18:40:17.282029+00:00 [running]> on host d88713c29571
[2024-12-26 18:41:46,609] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=assignment
AIRFLOW_CTX_TASK_ID=task_three
AIRFLOW_CTX_EXECUTION_DATE=2024-12-26T18:40:17.282029+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-12-26T18:40:17.282029+00:00
[2024-12-26 18:41:58,571] {logging_mixin.py:104} INFO - wrote file: titles.json
[2024-12-26 18:41:58,572] {python.py:151} INFO - Done. Returned value was: None
[2024-12-26 18:41:58,587] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=assignment, task_id=task_three, execution_date=20241226T184017, start_date=20241226T184146, end_date=20241226T184158
[2024-12-26 18:41:58,609] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-26 18:41:58,651] {local_task_job.py:151} INFO - Task exited with return code 0
